hack/testing/run_daemon_integration.sh

```diff
@@ -71,7 +71,8 @@ main() {
   integration::stop_pouchd
   integration::run_pouchd_background "${cmd}" "${flags}" "${pouchd_log}"
 
-  set +e; integration::ping_pouchd; code=$?; set -e
+  # use subshell to ping
+  set +e; ( integration::ping_pouchd ); code=$?; set -e
```

Comment:
why modify it?

Reply:
the `ping_pouchd` will exit which causes that it doesn't show the fail log.

Reply:
using subshell is to make sure that the following script can be executed.